tikv-client/src/main/java/com/pingcap/tikv/codec/MyDecimal.java

```diff
@@ -25,7 +26,7 @@
   // how many digits that a word has
   private static final int digitsPerWord = 9;
   // MyDecimal can holds at most 9 words.
-  private static final int wordBufLen = 9;
+  public static int wordBufLen = 9;
```

Comment:
You should state why you change wordBufLen into public and non-final. You could make a new MyDecimal via copy rather than changing its fields directly.